static/styles/portico/portico-signin.scss

```diff
@@ -832,15 +832,28 @@ button#register_auth_button_gitlab {
         }
     }
 
-    .help-box {
-        width: calc(100% - 25px);
+    #id_email {
+        display: inline-block;
+        font-weight: normal;
+        margin: 25px 2px 2px;
+        text-align: left;
+        width: 100%;
+        overflow-wrap: break-word;
```

Comment:
You usually don't put `id` into an id name, since it's redundant. Just looking at this diff, four properties seem redundant: `#id_email` is a div, which has `display: block` and `width: 100%;` by default. `font-weight: normal;` and `text-align: left;` should also be the default in general.

Reply:
`#id_email` is being used all over Zulip for email related elements and the disabled email input also had `id_email` as `id` so I didn't changed that. But we can change it to something like `#registration-email`.

`margin-top` property doesn't work on child elements of a `div` on using `display: block;` on the child element. Here is a [stackoverflow thread](https://stackoverflow.com/questions/1762539/margin-on-child-element-moves-parent-element) related to this. All other `input` fields are using `display: inline-block;` property to overcome this behaviour so I also used the same.

The default `font-wieght` is set to `300` in `body` which doesn't look good for displaying email. The `text-align` is set to `center` in the parent-div of the email-div.

Reply:
If we use it elsewhere it's better to keep it consistent. Can't we use `padding-top`?

Ah, didn't know we set a non-default font-weight. I think it would be better to set `text-align: center;` only there where we need it, so for the h1, the pitch and the button. This is probably best refactored in a separate commit. But since it was that way before, you don't need to change it now.